Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,188 @@
+
+                if (highestAmountOfPerson >= highestBalanceEver)
+                {
+                    persons = AddPerson(currentPerson, persons);
+                }
+            }
+
+            return persons;
+        }
+
+        private static Person[] AddPerson(Person person, Person[] persons)
+        {
+            var newArray = new Person[persons.Length + 1];
+            for (int i = 0; i < persons.Length; i++)
+            {
+                newArray[i] = persons[i];
+            }
+
+            newArray[^1] = person;
+            return newArray;
+        }
+
+        public static Person[] FindMostPoorPerson(string[] peopleAndBalances)
+        {
+            var parsedFile = BalanceParser.Parser(peopleAndBalances);
+            return FindPoorest(parsedFile);
+        }
+
+        internal static Person[] FindPoorest(Person[] balances)
+        {
+            CultureInfo.CurrentCulture = new CultureInfo(culture);
+            var persons = new Person[0];
+            var lowestAmount = decimal.MaxValue;
+
+            for (int i = 0; i <= balances.Length - 1; i++)
+            {
+                var currentPerson = balances[i];
+                var lowestAmountOfPerson = currentPerson.CurrentBalance();
+
+                if (lowestAmount > lowestAmountOfPerson)
+                {
+                    lowestAmount = lowestAmountOfPerson;
+
+                    persons = new Person[0];
+                }
+
+                if (lowestAmount >= lowestAmountOfPerson)
+                {
+                    persons = AddPerson(currentPerson, persons);
+                }
+            }
+            return persons;
+        }
+
+        public static Person[] FindPersonWithBiggestLoss(string[] peopleAndBalances)
+        {
+            var parsedFile = BalanceParser.Parser(peopleAndBalances);
+
+            return FindBiggestLoss(parsedFile);  
+        }
+
+        private static Person[] FindBiggestLoss(Person[] balances)
+        {
+            CultureInfo.CurrentCulture = new CultureInfo(culture);
+            var persons = new Person[0];
+            var biggestLoss = decimal.MinValue; 
+
+            for (int i = 0; i <= balances.Length - 1; i++)
+            {
+                var currentPerson = balances[i]; 
+
+                if (currentPerson.GetBalance().Length <= 1)
+                {
+                    return new Person[0];
+                }
+
+                 var biggestLossPerson = currentPerson.BiggestLoss(); 
+                
+                if (biggestLossPerson > biggestLoss)
+                {
+                    biggestLoss = biggestLossPerson;
+                    persons = new Person[0]; 
+                }
+
+                if (biggestLossPerson >= biggestLoss)
+                {
+                    persons = AddPerson(currentPerson, persons);
+                }
+            }
+
+            return persons;
+        }
+
+
+        private static decimal FindBiggestLossPerson(decimal[] allAmounts)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'FindBiggestLossPerson'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)